install.md

```diff
@@ -60,30 +60,40 @@ There are two different ways of creating a custom image. The first one consists
 
 The following steps can be followed in order to create a custom image:
 
-    1. Create a toolbox - in this case I'm using a custom image as base
+1. Create a toolbox - in this case I'm using a custom image as base
 
-        ```
-        $ toolbox create -i docker.io/akdev1l/ubuntu-toolbox:22.04
-        Created container: ubuntu-toolbox-22.04
-        Enter with: toolbox enter ubuntu-toolbox-22.04
-        ```
+```console
```

Comment:
The triple backtick code blocks need to be indented.  Otherwise the numbered list will keep getting reset.  See:
![image](https://github.com/containers/containertoolbx.org/assets/3208107/08aa89fd-0c1b-43d5-bd1b-cc4ee1e69943)

Reply:
I took the liberty to add the indentation so that the numbers don't get reset.

Reply:
Ok, feel free to update if necessary